### ISSUE TITLE
Help for custom fields

### ISSUE THREAD

**Issue by WillianSilvaDonz** [zendesk/zendesk_api_client_php#333]
Good afternoon I wanted to know if anyone knows how to implement custom fields.

**Comment by WillianSilvaDonz** [comment]
I need to update a custom field

**Comment by nbittmannzendesk** [comment]
Try something like this and make sure you're using the correct field id


`<?php


// load Composer
require 'vendor/autoload.php';

use Zendesk\API\HttpClient as ZendeskAPI;

$subdomain = "{SubDomain}";
$username  = "{User}"; // replace this with your registered email
$token     = "{Token}"; // replace this with your token

$client = new ZendeskAPI($subdomain);
$client->setAuth('basic', ['username' => $username, 'token' => $token]);

// Update a ticket
$client->tickets()->update({Ticket_ID},[
  'status' => 'solved',
  'custom_fields' => [
      'id' => '28259501',
        'value' => '1234'
      ],
    'comment'  => [
      'body' => 'Update to solved.'
        ]
          ]);
?>`

**Comment by jmramos02** [comment]
@WillianSilvaDonz do you still need help on this issue?

**Comment by firxworx** [comment]
I posted a gist for a php example of creating a ticket with multiple custom fields: 
https://gist.github.com/firxworx/db816b95e8bd18b84a3a684467a7fe25

At least for creating a ticket (slightly different vs your example of updating a ticket) the syntax appears incorrect. I tried that too for the heck of it and my custom field wasn't getting set in zen. I think you'd need to wrap your array w/ 'id' and 'value' in another array for it to work (or at least you do for the create ticket case).

(note I haven't reviewed the API client code)
